pinot-spi/src/main/java/org/apache/pinot/spi/config/table/SegmentsValidationAndRetentionConfig.java

```diff
@@ -38,6 +38,8 @@
   private String _schemaName;
   private String _timeColumnName;
   private TimeUnit _timeType;
+  // Flag to indicate if null value in time column should be handled.
+  private boolean _nullTimeColumnHandlingEnabled;
```

Comment:
Suggest renaming to `_allowNullTimeValue`

Reply:
+1